framework_tvb/tvb/adapters/simulator/simulator_fragments.py

```diff
@@ -168,9 +168,10 @@ def __init__(self, prefix='', project_id=None):
 
 class SimulatorFinalFragment(ABCAdapterForm):
 
-    def __init__(self, prefix='', project_id=None):
+    def __init__(self, prefix='', project_id=None, simulation_number=1):
         super(SimulatorFinalFragment, self).__init__(prefix, project_id)
-        self.simulation_name = ScalarField(Attr(str, doc='Name for the current simulation configuration',
+        default_simulation_name = "default_simulation_" + str(simulation_number)
```

Comment:
I would put here "simulation_"+count, to match the previous names:
https://www.thevirtualbrain.org/tvb/zwei/image-zoom/122365/27?stage=brainsimulator